packages/bus-rdf-join/lib/ActorRdfJoin.ts

```diff
@@ -18,15 +18,22 @@ import {EmptyIterator} from "asynciterator";
 export abstract class ActorRdfJoin extends Actor<IActionRdfJoin, IMediatorTypeIterations, IActorQueryOperationOutput> {
 
   /**
-   * Can be used by subclasses to indicate the max number of streams that can be joined.
+   * Can be used by subclasses to indicate the max or min number of streams that can be joined.
    * 0 for infinity.
+   * By default, this indicates the max number, but can be inverted by setting limitEntriesMin to true.
    */
-  protected maxEntries: number;
+  protected limitEntries: number;
+  /**
+   * If true, the limitEntries field is a lower limit,
+   * otherwise, it is an upper limit.
+   */
+  protected limitEntriesMin: boolean;
 
   constructor(args: IActorArgs<IActionRdfJoin, IMediatorTypeIterations, IActorQueryOperationOutput>,
-              maxEntries?: number) {
+              limitEntries?: number, limitEntriesMin?: boolean) {
```

Comment:
since a new field gets added, why not add a min field and a max field instead of 1 field and a boolean? Gives more flexibility in the future.

Reply:
True. Let's revisit this should the need for it arise.